pilot/pkg/security/authn/v1alpha1/policy_applier.go

```diff
@@ -400,7 +401,7 @@ func (a v1alpha1PolicyApplier) InboundFilterChain(sdsUdsPath string, sdsUseTrust
 				TLSContext:       tls,
 				ListenerFilters: []ldsv2.ListenerFilter{
 					{
-						Name:       EnvoyTLSInspectorFilterName,
```

Comment:
remove this unused const ?

Reply:
Yes, this is same as envoy TlsInspector

Reply:
I mean remove this var declaration from this file(and test file:)

Reply:
oh, sorry. I thought I removed.